gitlab/client.py

```diff
@@ -16,7 +16,9 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """Wrapper for the GitLab API."""
 
+import os
 import time
+from argparse import Namespace
```

Comment:
nit: you know me I prefer `import argparse` as I like to read the code and see `argparse.Namespace` 😊

Reply:
Thanks for this comment. After I looked at this, I realized I should probably decouple any `argparse` code from `client.py` since that's supposed to be independent of the CLI. Maybe we could just pass a dict of key-values to it, and do the conversion from `cli.py`.

This way people could even do something like this, if they want programmatically use config files but override something..
`gl = gitlab.Gitlab.merge_config({"private_token": "glpat-sometoken"}, gitlab_id="my-server", ["some-file.cfg"])`

Not sure, but some might find that useful :grinning: WDYT?

Reply:
That sounds reasonable to me. Makes sense that this file doesn't need to be worries about argument parsing. But not sure how much work that is for you...

Reply:
Converted to a dict and using `vars`() in `cli.py` now.

Was even toying with the idea of using a `kwargs` dict and just let people do `gitlab.Gitlab.merge_config(private_token="glpat-bla")` which would be quite clean. Maybe I'll add that later.